extensions/wikia/CommunityPage/tests/CommunityPageSpecialTopAdminsFormatterTest.php

```diff
@@ -12,7 +12,6 @@ public function setUp() {
 	 */
 	public function testAdminsDataFormat( $param, $expected, $expectedTopAdminsCount, $message ) {
 		$result = CommunityPageSpecialTopAdminsFormatter::prepareData( $param );
-		$this->assertEquals( $result['otherAdminsCount'], $expected['otherAdminsCount'], "$message 'otherAdminsCount' field doesn't match" );
 		$this->assertEquals( $result['haveOtherAdmins'], $expected['haveOtherAdmins'], "$message 'haveOtherAdmins' field doesn't match" );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "haveOtherAdmins" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)